### ISSUE TITLE
Remove additional maven repos from pom and document instead

### ISSUE THREAD

**Issue by scrobbleme** [Communote/communote-server#2]
The problem occurs, when these repositories go offline or send errors for whatever reason, as the build may fail and you have to edit the pom, which is not nice for non-committers. This also may occur, when the packages are used as dependencies itself.

Much better: Add the needed repositories to the documentation and describe how to add them in settings.xml (or Nexus..)

**Comment by rwi** [comment]
@scrobbleme I wasn't sure whether it was good idea to add the repositories to the pom ;)
The advantage of having them in the pom is that the source can be built without further configuration. Moreover, the plugin repositories which are required for translating the REST API model can be restricted to the 'mda' profile and are therefore not checked when building without this profile.

The downside are the points you mentioned above. But if one of them occurs can't you just add a working repo (directly or as a mirror) to the settings.xml?

**Comment by scrobbleme** [comment]
@rwi Yes, it should be possible to create a mirror then. But better let them do it upfront, than let them figuring it out a solution later, when it happens.

Maybe just wait for now and see what happens ;)
